index.html

```diff
@@ -15,6 +15,7 @@
     <title>Envoy Mobile</title>
   </head>
   <body>
+    <div id="banner"><b>IMPORTANT</b> This project is in a pre-release stage, and is not available for production use.</div>
```

Comment:
```suggestion
    <div id="banner"><b>IMPORTANT</b> This project is in a pre-release stage. All features are not implemented and it is not ready for production use.</div>
```

Reply:
> All features are not implemented

IMO this reads a little oddly. Maybe `Not all features are implemented` or `No features are implemented`? (Not quite sure what the intent was)

Reply:
Sure +1 "Not all features are implemented..."

Reply:
```
This project is in a pre-release stage. Not all features are implemented and it is not ready for production use.
```
?

Reply:
🚢